design/user-profile.md

```diff
@@ -128,6 +164,14 @@ Built-in validators should cover:
 
 There will be a user attribute SPI allowing custom validators to be created.
 
+### Processors (optional)
```

Comment:
Would "Converter" be a better name? Processor makes me think that it does more than just to conversions.

Reply:
we will use Converter from now on